### DISCUSSION TITLE
Shader warning - Fails to build - Nature Shaders/Tree Leaves _Translucency

### DISCUSSION THREAD

**Discussion by hipicrit** [visualdesigncafe/nature-renderer#59] [Help] [answered]
Shader warning in 'Nature Shaders/Tree Leaves': '_TRANSLUCENCY': macro redefinition. Previous definition found at /:32. at /(19575)

The above is one of the errors I have been receiving when trying to build and run. I have also gotten this for
Nature Shaders/Grass and Nature Shaders/Tree Bark. I am able to build if I click in the top right of inspector 
and reset each shader, however it will pop up when I try to build again. I only have a simple scene with 100 X 100
terrain, few grasses, rocks, and trees. This is a test scene prior to expanding. Any ideas?

**Comment by maxplooi** [comment, marked as answer]
It is a simple shader warning that should not cause any issues and should not cause any build to fail. It is expected to happen and the warnings are ignored by the compiler. Do you get any other error that specifically causes the build to fail?

**Reply by hipicrit** [reply]
It may have been related to a prefab I was attempting to use or another asset I had imported. My troubleshooting level is still quite low so I started a new scene. I used some of the included prefabs such as rocks and it works brilliantly. Thanks for your reply.
